tests/check_consistent.py

```diff
@@ -68,17 +68,9 @@ def check_stubs():
                 else:
                     assert name.isidentifier(), f"Bad file name '{entry}' in stubs"
             else:
-                if entry in ("@python2", "@tests"):
+                if entry in ("@tests",):
```

Comment:
Any reason not to do this?

```suggestion
                if entry == "@tests":
```